### ISSUE TITLE
Curious about the loss function

### ISSUE THREAD

**Issue by happyseone** [White-Link/UnsupervisedScalableRepresentationLearningTimeSeries#7]
Hi, I'm sorry to have one more question again.

I saw the loss function and it does not seem like a normal triplet loss.
To me, it is more like kind of  crossentropy loss.
![20200407_224607](https://user-images.githubusercontent.com/18029954/78736835-15b6bf80-7989-11ea-856a-ce2022a326e0.png)
Could you explain why you make this formula of loss function?(dot product and sigmoid, etc)

**Comment by martinjaggi** [comment]
this is the same loss function as used in word2vec, but on top of our encoder f. you find it's discussion then you google negative sampling / Noise Contrastive Estimation, see for example here https://ruder.io/word-embeddings-softmax/
